### ISSUE TITLE
[Bugs] ADF - games issues

### ISSUE THREAD

**Issue by nemo93** [midwan/amiberry#152]
The following games failed to properly run. If a separate ticket per game is the prefered way please let me know and I'll fill a ticket in for each. 

**Read Me first:** 
For WHDLoad related issues please consider posting on thread #122 instead.

-----------------------------------------------------

**Total games listed below: 16
(B) Non-working/unplayable: 7
(G) Working/playable yet w glitches: 9**

-----------------------------------------------------

* **(B) Bombuzal [version (1988)(Image Works)[cr QTX][t +3 Vector]] - ECS/OCS
=> crash just before you start playing level 1.
[uae for Bombuzal](https://github.com/midwan/amiberry/files/1293649/BarbarianPalace.txt)

* **(G) Cannon Fodder [v1.0 (25.01.2001)]** - ECS/OCS
=> game runs fine. However at the very first intro mission screen where you can see a chopper/helicopter hovering over jungle, you could see that the tail/rotor is not there. This bug can be seen only on this first intro screen, on the following missions tail is there no problem.
[my uae for Cannon Fodder](https://github.com/midwan/amiberry/files/1290625/Cannon.Fodder.txt)

* **(G) Dungeon Master 2 [Version 1.0 (03.09.2003)]** - AGA
=> game runs fine. However the loading screen is flickering.
[my uae for Dungeon Master 2 AGA](https://github.com/midwan/amiberry/files/1290624/Dungeon.Master.2.txt)

* **(G) New York Warriors [version 1.4 (07.10.17)]** - ECS/OCS
=> game does run absolutely fine however there's no sound effect. To be accurate you can hear sound effects but only once in a while (once every minute or so without any clear patterns of when sounds would be heard).
[my uae for New York Warriors](https://github.com/midwan/amiberry/files/1282803/New.York.Warriors.txt)

* **(G) Panza Kick Boxing [version 1.6 (01.07.06)]** - ECS/OCS
=> runs ok yet glitches appear on the boxers characters during an animation/move. Just launch a fight against any opponent, do some moves and the glitches would appear pretty quickly.
[my uae for Panza Kick Boxing](https://github.com/midwan/amiberry/files/1282798/Panza.Kick.Boxing.txt)

* **(B) Second Samurai [version 1.1 (01-Feb-2001)]** - AGA
=> graphical mess. Logo and loading screens are fine but as soon as you enter the main menu screen it's messy. It's a bit hard to describe but clearly the colors are not where they are supposed to be! **Please do note that the ECS/OCS version of this game is working like a charm.**
[my uae for Second Samurai AGA](https://github.com/midwan/amiberry/files/1282813/Second.Samurai.txt)

* **(G) Shadow Fighter [(Version 1.0 (24.03.2007))]** - AGA
=> runs ok yet too slow (compare with the ECS version to see what the real speed should be). **Please do note that the ECS/OCS version of this game is working like a charm.** Thanks to @vultureg7 and to @ransom1122 for reporting and confirming that one (original ticket #138).
[my uae for Shadow Fighter](https://github.com/midwan/amiberry/files/1293641/Body.Blows.txt)

* **(G) Striker [(version 1.0 (01.11.2001))]** - ECS/OCS
=> runs ok yet character sprites flicker during the game. Problem occurs on both WHDLoad & ADF versions of the game. Thanks to @ransom1122 for reporting and confirming that one (original ticket #146).
[my uae for Striker](https://github.com/midwan/amiberry/files/1356029/Unreal.txt)

* **(G) Unreal [Version 1.2 (11.03.2008)]** - ECS/OCS
=> runs ok yet glitches appear during the first stage while you're flying the dragon. Each time there's an indication (turn left/right etc) a glitch can be seen instead of the arrow.
[my uae for Unreal](https://github.com/midwan/amiberry/files/1356029/Unreal.txt)

* **(B) War Zone (Core Design) [version 2.0 (11-apr-2017)]** - ECS/OCS
=> I do confirm the issue described in #113 by @ransom1122 
[my uae for War Zone (Core Design)](https://github.com/midwan/amiberry/files/1282854/War.Zone.Core.Design.txt)

* **(G) Worms Directors Cut [version 1.2 (21.08.2011)]** - AGA
=> UPDATE: game runs fine. However the loading screen is flickering.
[my uae for Worms Directors Cut AGA](https://github.com/midwan/amiberry/files/1290623/Worms.Directors.Cut.txt)

* **(G) XP8 [version 1.2 (02.01.03)]** - AGA
=> Text menu on the intro screen is garbled. **Please do note that the ECS/OCS version of this game is working like a charm.**
[my uae for XP8 AGA](https://github.com/midwan/amiberry/files/1356086/XP8.txt)

* **(B) Zool [version 2.0 (13-jan-08)]** - AGA
=> graphical mess. Same as Second Samurai. Logo/main screens are fine yet intro and game itself have messed up graphics. Still hard to describe but it's like colors are not properly rendered.
[my uae for Zool AGA](https://github.com/midwan/amiberry/files/1282827/Zool.txt)

* **(B) Zool 2 [Version 1.0 (28.05.2002)]** - AGA
=> graphical mess. Same as Second Samurai and Zool. Logo/main screens are fine yet game itself have messed up graphics. Still hard to describe but it's like colors are not properly rendered.
[my uae for Zool 2 AGA](https://github.com/midwan/amiberry/files/1282838/Zool.2.txt)
